packages/wonder-blocks-clickable/src/components/clickable-behavior.ts

```diff
@@ -175,7 +175,7 @@ type DefaultProps = {
 };
 
 export type ChildrenProps = {
-    onClick: (e: React.MouseEvent) => unknown;
+    onClick: (e: React.SyntheticEvent) => unknown;
```

Comment:
This is to align with `onClick` in a different object type that interacts with `ChildrenProps`.